ablySpec/RealtimeClientConnection.swift

```diff
@@ -782,7 +782,7 @@ class RealtimeClientConnection: QuickSpec {
                             delay(1.0) { done() }
                         }
 
-                        client.simulateLostConnection()
+                        client.simulateLostConnectionAndState()
```

Comment:
Ah I see this is fixed in this PR so ignore my previous comment @ricardopereira, just working my way through the backlog...